.github/workflows/ah.yml

```diff
@@ -0,0 +1,29 @@
+name: Update AH strategy rewards proof
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 0" # once a week on Sunday at midnight
+
+jobs:
+  rebase:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: |
+          yarn install
+          yarn hardhat compile
+
+      - run: |
+          yarn hardhat ahProofUpdate --network mainnet-prod
+        env:
+          DEPLOYER_PK: ${{ secrets.DEPLOYER_PK }}
+
+      - run: |
+          curl -X POST -H "Content-Type: application/json" "$DISCORD_WEBHOOK_URL" \
+            -d '{"content": "XUSD: Alpha Homera strategy rewards proof updated"}'
```

Comment:
typo: `Alpha Homora` like the Harry Potter spell:
![image](https://user-images.githubusercontent.com/96358263/158292153-d5077f93-9eec-4228-b4d7-4d4978ec805b.png)